conf/defaults.ini

```diff
@@ -1198,9 +1198,12 @@ lokiQueryBuilder = true
 # Experimental Explore to Dashboard workflow
 explore2Dashboard = true
 
-# Experimental Command Palette
+# Command Palette
 commandPalette = true
 
+# Experimental Explore Mixed Datasource
+exploreMixedDatasource = false
```

Comment:
FYI this isn't needed, and I think we should keep the default config clean. Feature flags are disabled by default, so no need to default it here again.

Reply:
Fixed!